### PR TITLE
[ru] Second set of terms for Russian localization

### PR DESCRIPTION
**Pull request by kirkonru**
### Description

This PR adds a second set of localized Russian terms.

In total, 14 terms were translated from English and 2 Russian-specific terms were added.

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/ru/native.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: Abstraction
```
Comment (kirkonru) [inline content/ru/native.md:2]:
```suggestion
title: Нативный
```

**Comment by kirkonru** [comment]
@shurup please review this one as well 😇

File: content/ru/blue-green-deployment.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+status: Completed
+category: concept
+tags: ["methodology", "application", ""]
+---
+
+## Описание
+
+Сине-зеленое развертывание — стратегия обновления работающих компьютерных систем с минимальным временем простоя.
+Оператор использует два окружения — синее и зеленое.
+Одно из них обслуживает production-трафик (в нем работает старая версия), а другое в это время обновляется.
```
Comment (shurup) [inline content/ru/blue-green-deployment.md:12]:
```suggestion
Одно из них обслуживает production-трафик (в нем работает текущая версия, с которой работают пользователи), а другое в это время обновляется.
```

File: content/ru/blue-green-deployment.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+После успешного завершения тестирования зеленого окружения (с новой версией) 
+production-трафик переключается на него (часто с помощью балансировщика нагрузки).
+Отметим, что в этой стратегии окружения обычно переключаются целиком, сразу со всеми [сервисами](/service/).
+Однако иногда этот термин используется по отношению к отдельным сервисам в системе.
+Чтобы избежать подобной двусмысленности, для отдельных компонентов предпочтительно 
+использовать термин "развертывание с нулевым временем простоя".
```
Comment (shurup) [inline content/ru/blue-green-deployment.md:18]:
```suggestion
использовать термин "развертывание с нулевым временем простоя" _(zero-downtime deployment)_.
```

File: content/ru/canary-deployment.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
+---
+title: Канареечное развертывание
+status: Completed
+category: concept
+tags: ["methodology", "application", ""]
+---
+
+## Описание
+
+Стратегия канареечных развертываний начинается с двух окружений:
```
Comment (shurup) [inline content/ru/canary-deployment.md:10]:
```suggestion
Стратегия канареечных развертываний _(canary deployments)_ начинается с двух окружений: 
```

File: content/ru/canary-deployment.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+title: Канареечное развертывание
+status: Completed
+category: concept
+tags: ["methodology", "application", ""]
+---
+
+## Описание
+
+Стратегия канареечных развертываний начинается с двух окружений: 
+одно из них обслуживает живой трафик, другое содержит обновленный код (но без трафика).
```
Comment (shurup) [inline content/ru/canary-deployment.md:11]:
```suggestion
одно из них обслуживает живой трафик, другое — содержит обновленный код (но без трафика).
```

File: content/ru/chaos-engineering.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
+---
+title: Хаос-инженерия
+status: Completed
+category: concept
+tags: ["methodology", "", ""]
+---
+
+## Описание
+
+Хаос-инженерия (Chaos Engineering или CE) — подход, при котором над [распределенной](/distributed-systems/) production-системой проводится ряд экспериментов,
```
Comment (shurup) [inline content/ru/chaos-engineering.md:10]:
```suggestion
Хаос-инженерия (_chaos engineering_, CE) — подход, при котором над [распределенной](/distributed-systems/) production-системой проводится ряд экспериментов, 
```

File: content/ru/chaos-engineering.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+---
+
+## Описание
+
+Хаос-инженерия (Chaos Engineering или CE) — подход, при котором над [распределенной](/distributed-systems/) production-системой проводится ряд экспериментов, 
+чтобы убедиться, что она способна противостоять турбулентным и неожиданным ситуациям.
+
+## Проблема
+
+Практики [SRE](/site-reliability-engineering/) и [DevOps](/devops/) концентрируются на
```
Comment (shurup) [inline content/ru/chaos-engineering.md:15]:
```suggestion
Практики [SRE](/site-reliability-engineering/) и [DevOps](/ru/devops/) концентрируются на 
```

File: content/ru/auto-scaling.md [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+Автомасштабирование обычно ассоциируется с облачными managed-сервисами, поскольку те предлагают массу реализаций этой функции 
+и значительно опережают большинство локальных (on-premise) развертываний по этому показателю.
+
+Ранее инфраструктура и приложения проектировались с учетом пиковой загрузки системы.
+Такой подход означал, что значительное количество ресурсов недоиспользуется, 
+а сама система неэлеастично реагировала на изменение числа пользователей и их потребностей.
+Неэластичность приводила к росту затрат и потерям от перебоев в работе во времена большого наплыва пользователей, 
+когда возможностей системы просто не хватало на всех.
+
+Используя облачные технологии, [виртуализацию](/virtualization/) и [контейнеризацию](/containerization/) приложений и их зависимостей,
```
Comment (shurup) [inline content/ru/auto-scaling.md:20]:
```suggestion
Используя облачные технологии, [виртуализацию](/virtualization/) и [контейнеризацию](/ru/containerization/) приложений и их зависимостей, 
```

File: content/ru/cluster.md [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+category: Concept
+tags: ["infrastructure", "fundamental", ""]
+---
+
+## Описание
+
+Кластер — группа компьютеров или приложений, которые объединены общей целью.
+В контексте нативных облачных вычислений этот термин чаще всего применяется к [Kubernetes](/kubernetes/).
+Кластер Kubernetes — это набор сервисов (или рабочих нагрузок), которые выполняются в контейнерах (обычно на разных машинах).
+Совокупность всех этих [контейнеризованных](/containerization/) сервисов, соединенных по сети, представляет собой кластер.
```
Comment (shurup) [inline content/ru/cluster.md:13]:
```suggestion
Совокупность всех этих [контейнеризованных](/ru/containerization/) сервисов, соединенных по сети, представляет собой кластер.
```

File: content/ru/container-image.md [hunk 10]
```diff
@@ -0,0 +1,29 @@
...
+category: concept
+tags: ["", "", ""]
+---
+
+## Описание
+
+Образ контейнера — это неизменяемый статический файл, содержащий зависимости, необходимые для создания [контейнера](/container/).
+Эти зависимости могут включать один исполняемый двоичный файл, 
+системные библиотеки, системные инструменты, переменные окружения и другие необходимые настройки платформы.
+Образы контейнеров являются результатом [контейнеризации](/containerization/) приложения и обычно хранятся в реестрах контейнеров,
```
Comment (shurup) [inline content/ru/container-image.md:13]:
```suggestion
Образы контейнеров являются результатом [контейнеризации](/ru/containerization/) приложения и обычно хранятся в реестрах контейнеров, 
```

File: content/ru/client-server-architecture.md [hunk 11]
```diff
@@ -0,0 +1,37 @@
...
+
+В клиент-серверной архитектуре логика (или код), составляющая приложение, распределяется между двумя или более компонентами: 
+клиентом, запрашивающим выполнение некой работы 
+(например, веб-приложение Gmail, работающее в браузере), 
+и одним или несколькими серверами, удовлетворяющими этот запрос 
+(например, сервис отправки почты, работающий на компьютерах Google в облаке).
+В этом примере исходящие письма отправляются клиентом (веб-приложением, работающим в браузере) 
+на сервер (компьютеры Gmail, которые пересылают исходящие письма их получателям).
+
+Этот подход отличается от автономных приложений (например, настольных), которые выполняют всю работу в одном месте.
```
Comment (shurup) [inline content/ru/client-server-architecture.md:18]:
```suggestion
Этот подход отличается от автономных приложений (например, десктопных), которые выполняют всю работу в одном месте.
```

File: content/ru/client-server-architecture.md [hunk 12]
```diff
@@ -0,0 +1,37 @@
...
+## Решение
+
+Реализуя логику приложения на удаленном сервере или сервисе, операторы могут обновлять ее, 
+не изменяя логику на стороне клиента. 
+Поэтому обновления можно производить гораздо чаще.
+Хранение данных на сервере позволяет многим клиентам видеть одни и те же данные и использовать их совместно.
+Рассмотрим разницу между онлайновым текстовым процессором и традиционным офлайновым, который работает на локальном компьютере.
+В первом случае ваши файлы находятся на стороне сервера и 
+могут быть доступны другим пользователям, которые просто скачивают их с сервера.
+В втором случае (когда интернет не было доступен повсеместно) файлы приходилось копировать
```
Comment (shurup) [inline content/ru/client-server-architecture.md:36]:
```suggestion
Во втором случае (когда интернет не было доступен повсеместно) файлы приходилось копировать 
```

File: content/ru/cloud-computing.md [hunk 13]
```diff
@@ -0,0 +1,24 @@
+---
+title: Облачные вычисления
+status: Completed
+category: concept
+tags: ["infrastructure", "fundamental", ""]
+---
+
+## Описание
+
+"Облачные вычисления" (cloud computing) — собирательное название систем, которые предоставляют вычислительные ресурсы (процессорные, сетевые и дисковые) по запросу через сеть Интернет. В резульатет пользователи получают доступ к вычислительным мощностям, которые физически расположены в совершенно другом, удаленном месте.
```
Comment (shurup) [inline content/ru/cloud-computing.md:10]:
```suggestion
Облачные вычисления _(cloud computing)_ — собирательное название систем, которые предоставляют вычислительные ресурсы (процессорные, сетевые и дисковые) по запросу через сеть Интернет. В результате пользователи получают доступ к вычислительным мощностям, которые физически расположены в совершенно другом, удаленном месте.
```

File: content/ru/cloud-native-security.md [hunk 14]
```diff
@@ -0,0 +1,34 @@
+---
+title: Нативная облачная безопасность
+status: Completed
+category: concept
+tags: ["security", "", ""]
+---
+
+## Описание
+
+Нативная облачная безопасность — подход, при котором безопасность встраивается в [нативные облачные приложения](/cloud-native-apps/).
```
Comment (shurup) [inline content/ru/cloud-native-security.md:10]:
```suggestion
Нативная облачная безопасность — подход, при котором безопасность встраивается в [нативные облачные приложения](/ru/cloud-native-apps/).
```

File: content/ru/cloud-native-security.md [hunk 15]
```diff
@@ -0,0 +1,34 @@
...
+В его рамках безопасность становится частью всего жизненного цикла приложения — от разработки до production.
```
Comment (shurup) [inline content/ru/cloud-native-security.md:11]:
```suggestion
В его рамках безопасность становится частью всего жизненного цикла приложения: от разработки до production.
```

File: content/ru/cloud-native-security.md [hunk 16]
```diff
@@ -0,0 +1,34 @@
...
+Нативная облачная безопасность сред направлена на обеспечение тех же стандартов, что и традиционные модели безопасности, 
+но при этом адаптирована к особенностям облачных окружений, 
+а именно, к быстрым изменениям кода и эфемерной инфраструктуре.
+Нативная облачная безопасность тесно связана с практикой, известной как [DevSecOps](/devsecops/).
```
Comment (shurup) [inline content/ru/cloud-native-security.md:15]:
```suggestion
Нативная облачная безопасность тесно связана с практикой, известной как [DevSecOps](/ru/devsecops/).
```

File: content/ru/cloud-native-security.md [hunk 17]
```diff
@@ -0,0 +1,34 @@
...
+
+## Проблема
+
+Традиционные модели безопасности строились с учетом ряда допущений, которые больше не актуальны.
+Нативные облачные приложения часто меняются, используют большое количество инструментов и библиотек с открытым исходным кодом, 
+работают в инфраструктуре, контролируемой вендором, и подвержены быстрым изменениям инфраструктуры.
+Проверка кода, длительные циклы обеспечения качества, сканирование уязвимостей на хостах 
+и проверки безопасности "последней минуты" не соответствуют потребностям нативных облачных приложений.
```
Comment (shurup) [inline content/ru/cloud-native-security.md:23]:
```suggestion
и проверки безопасности в самый последний момент не соответствуют потребностям нативных облачных приложений.
```

File: content/ru/cloud-native-security.md [hunk 18]
```diff
@@ -0,0 +1,34 @@
...
+но при этом адаптирована к особенностям облачных окружений, 
+а именно, к быстрым изменениям кода и эфемерной инфраструктуре.
+Нативная облачная безопасность тесно связана с практикой, известной как [DevSecOps](/devsecops/).
+
+## Проблема
+
+Традиционные модели безопасности строились с учетом ряда допущений, которые больше не актуальны.
+Нативные облачные приложения часто меняются, используют большое количество инструментов и библиотек с открытым исходным кодом, 
+работают в инфраструктуре, контролируемой вендором, и подвержены быстрым изменениям инфраструктуры.
+Проверка кода, длительные циклы обеспечения качества, сканирование уязвимостей на хостах
```
Comment (shurup) [inline content/ru/cloud-native-security.md:22]:
```suggestion
Рецензии кода, длительные циклы обеспечения качества, сканирование уязвимостей на хостах 
```

File: content/ru/cloud-native-security.md [hunk 19]
```diff
@@ -0,0 +1,34 @@
...
+## Решение
+
+Нативная облачная безопасность — новый подход к защите приложений, 
+предполагающий переход от традиционных моделей безопасности к таким, 
+в которых безопасность участвует в каждом этапе релизного цикла.
+Ручные аудиты и проверки в значительной степени заменяются автоматизированным сканированием.
+Пайплайны быстрого релиза кода интегрированы с инструментами, которые сканируют код на наличие уязвимостей до его компиляции.
+Библиотеки с открытым исходным кодом берутся из надежных источников и отслеживаются на предмет наличия уязвимостей.
+Вместо того чтобы замедлять изменения, модель нативной облачной безопасности способствует им, 
+постоянно обновляя уязвимые компоненты или следя за тем, чтобы инфраструктура регулярно заменялась.
```
Comment (shurup) [inline content/ru/cloud-native-security.md:34]:
```suggestion
постоянно обновляя уязвимые компоненты или следя за тем, чтобы инфраструктура регулярно обновлялась.
```

File: content/ru/cluster.md [hunk 20]
```diff
@@ -0,0 +1,28 @@
...
+title: Кластер
+status: Completed
+category: Concept
+tags: ["infrastructure", "fundamental", ""]
+---
+
+## Описание
+
+Кластер — группа компьютеров или приложений, которые объединены общей целью.
+В контексте нативных облачных вычислений этот термин чаще всего применяется к [Kubernetes](/kubernetes/).
```
Comment (shurup) [inline content/ru/cluster.md:11]:
```suggestion
В контексте нативных облачных вычислений этот термин чаще всего применяется к [Kubernetes](/ru/kubernetes/).
```

File: content/ru/cluster.md [hunk 21]
```diff
@@ -0,0 +1,28 @@
...
+Кластер Kubernetes — это набор сервисов (или рабочих нагрузок), которые выполняются в контейнерах (обычно на разных машинах).
+Совокупность всех этих [контейнеризованных](/containerization/) сервисов, соединенных по сети, представляет собой кластер.
+
+## Проблема 
+
+Программное обеспечение, работающее на одном компьютере, представляет собой единую точку отказа
```
Comment (shurup) [inline content/ru/cluster.md:17]:
```suggestion
Программное обеспечение, работающее на одном компьютере, представляет собой единую точку отказа: 
```

File: content/ru/cluster.md [hunk 22]
```diff
@@ -0,0 +1,28 @@
...
+
+Кластер — группа компьютеров или приложений, которые объединены общей целью.
+В контексте нативных облачных вычислений этот термин чаще всего применяется к [Kubernetes](/kubernetes/).
+Кластер Kubernetes — это набор сервисов (или рабочих нагрузок), которые выполняются в контейнерах (обычно на разных машинах).
+Совокупность всех этих [контейнеризованных](/containerization/) сервисов, соединенных по сети, представляет собой кластер.
+
+## Проблема 
+
+Программное обеспечение, работающее на одном компьютере, представляет собой единую точку отказа 
+— если этот компьютер сломается или кто-то случайно отключит его питание,
```
Comment (shurup) [inline content/ru/cluster.md:18]:
```suggestion
если этот компьютер сломается или кто-то случайно отключит его питание, 
```

File: content/ru/cluster.md [hunk 23]
```diff
@@ -0,0 +1,28 @@
...
+— если этот компьютер сломается или кто-то случайно отключит его питание, 
+критически важная для бизнеса система может быть выведена из строя.
+Именно поэтому современное программное обеспечение, как правило, строится в виде [распределенных приложений](/distributed-apps/), объединенных в кластеры.
+
+## Решение
+
+Распределенные приложения, объединенные в кластеры, работают на множестве вычислительных машин — в этом случае единая точка отказа не образуется.
+Но создавать распределенные системы непросто. 
+На самом деле этим занимается отдельная дисциплина в области компьютерных наук.
+Потребность в глобальных системах и годы проб и ошибок привели к созданию нового вида технологического стека — [нативных облачных технологий](/cloud-native-tech/).
```
Comment (shurup) [inline content/ru/cluster.md:27]:
```suggestion
Потребность в глобальных системах и годы проб и ошибок привели к созданию нового вида технологического стека — [нативных облачных технологий](/ru/cloud-native-tech/).
```

File: content/ru/container-image.md [hunk 24]
```diff
@@ -0,0 +1,29 @@
+---
+title: Образ контейнера
+status: Feedback Appreciated
+category: concept
+tags: ["", "", ""]
+---
+
+## Описание
+
+Образ контейнера — это неизменяемый статический файл, содержащий зависимости, необходимые для создания [контейнера](/container/).
```
Comment (shurup) [inline content/ru/container-image.md:10]:
```suggestion
Образ контейнера — это неизменяемый статический файл, содержащий зависимости, необходимые для создания [контейнера](/ru/container/).
```

File: content/ru/container-image.md [hunk 25]
```diff
@@ -0,0 +1,29 @@
...
+Эти зависимости могут включать один исполняемый двоичный файл,
```
Comment (shurup) [inline content/ru/container-image.md:11]:
```suggestion
Эти зависимости могут включать один исполняемый бинарный файл, 
```

File: content/ru/container-image.md [hunk 26]
```diff
@@ -0,0 +1,29 @@
...
+
+Традиционно серверы настраиваются под каждое окружение, и на них развертываются приложения.
+Любое несоответствие конфигурации окружений — большая проблема, которая часто приводит к простоям или неудачным развертываниям.
+Окружение приложения должно быть повторяемым и строго определенным; 
+в противном случае возрастает вероятность возникновения ошибок, связанных с окружением.
+Если окружение настроено неадекватно или с ошибками, [горизонтальное](/horizontal-scaling/) и [вертикальное](/vertical-scaling/) масштабирование приложений становится проблематичным.
+
+## Решение
+
+Образы контейнеров содержат приложение с любыми его рантайм-зависимостями, например, с сервером этого приложения.
```
Comment (shurup) [inline content/ru/container-image.md:27]:
```suggestion
Образы контейнеров содержат приложение с любыми его runtime-зависимостями, например, с сервером этого приложения.
```

File: content/ru/containers-as-a-service.md [hunk 27]
```diff
@@ -0,0 +1,36 @@
...
+status: Deprecated
+category: Technology
+draft: true
+tags: ["platform", "", ""]
+---
+
+## Описание
+
+Контейнеры-как-услуга (Containers-as-a-Service, CaaS) — облачный сервис, 
+позволяющий управлять и развертывать приложения с использованием [контейнерной](/container/) [абстракции](/abstraction/).
```
Comment (shurup) [inline content/ru/containers-as-a-service.md:12]:
```suggestion
позволяющий управлять и развертывать приложения с использованием [контейнерной](/ru/container/) [абстракции](/ru/abstraction/).
```

File: content/ru/containers-as-a-service.md [hunk 28]
```diff
@@ -0,0 +1,36 @@
...
+Сервис можно развернуть как локально, так и в облаке.
+
+Поставщики CaaS предлагают фреймворк или платформу оркестрации, 
+автоматизирующую ключевые ИТ-функции, на базе которых развертываются и управляются контейнеры.
+Она помогает разработчикам создавать безопасные и [масштабируемые](/scalability/) контейнерные приложения.
+Приобретая только те ресурсы, которые им необходимы (планирование, балансировку нагрузки и т. д.), 
+пользователи экономят средства и повышают эффективность.
+Контейнеры позволяют создавать целостные окружения для быстрой разработки и 
+доставки [нативных облачных приложений](/cloud-native-apps/), которые могут работать где угодно.
```
Comment (shurup) [inline content/ru/containers-as-a-service.md:21]:
```suggestion
доставки [нативных облачных приложений](/ru/cloud-native-apps/), которые могут работать где угодно.
```

File: content/ru/containers-as-a-service.md [hunk 29]
```diff
@@ -0,0 +1,36 @@
...
+## Проблема
+
+Без CaaS командам разработчиков ПО приходится развертывать, управлять и мониторить 
+инфраструктуру, на которой работают контейнеры.
+
+## Решение
+
+Развертывая контейнерные приложения на CaaS-платформе, 
+пользователи получают возможность отслеживать производительность системы с помощью агрегации логов и средств мониторинга.
+CaaS также включает встроенную функциональность для [автоматического масштабирования](/auto-scaling/) и управления оркестрацией
```
Comment (shurup) [inline content/ru/containers-as-a-service.md:32]:
```suggestion
CaaS также включает встроенную функциональность для [автоматического масштабирования](/ru/auto-scaling/) и управления оркестрацией 
```

File: content/ru/containers-as-a-service.md [hunk 30]
```diff
@@ -0,0 +1,36 @@
...
+
+## Решение
+
+Развертывая контейнерные приложения на CaaS-платформе, 
+пользователи получают возможность отслеживать производительность системы с помощью агрегации логов и средств мониторинга.
+CaaS также включает встроенную функциональность для [автоматического масштабирования](/auto-scaling/) и управления оркестрацией 
+и позволяет командам создавать прозрачные и высокодоступные [распределенные системы](/distributed-systems/).
+Значиельно упрощая развертывание, CaaS повышает скорость разработки.
+В то время как контейнеры обеспечивают единообразие целей развертывания, 
+CaaS снижает операционные затраты за счет сокращения ресурсов [DevOps](/devops/), необходимых для управления развертыванием.
```
Comment (shurup) [inline content/ru/containers-as-a-service.md:36]:
```suggestion
CaaS снижает операционные затраты за счет сокращения ресурсов [DevOps](/ru/devops/), необходимых для управления развертыванием.
```

**Review by shurup — CHANGES_REQUESTED** [review]
Great job was already done here! Here are my corrections.

**Comment by shurup** [comment]
Based on our [recent Slack discussion](https://cloud-native.slack.com/archives/C05G46RMQTX/p1698057198264159?thread_ts=1698038167.603209&cid=C05G46RMQTX), closing this in favour of a bunch of separate PRs with each of them adding one term.

**Review by shurup — COMMENTED** [review]
- sorry, it was a wrong comment here -

**Pull request closed without merging**